### PR TITLE
[MC-32221] Update Elasticsearch/Zend/Apache for 2.3.5

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) updates versions for 2.3.5.

- Elasticsearch versions
- Zend to Laminas
- Couple Apache 2.2 refs.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/config-guide/elasticsearch/es-overview.html

### CODE REVIEW

File: src/guides/v2.3/config-guide/elasticsearch/es-downgrade.md [hunk 1]
```diff
@@ -8,32 +8,22 @@ functional_areas:
   - Setup
 ---
 
-The Magento 2.3.1 update adds support for Elasticsearch (ES) 6.x.
-Magento still provides connectivity for ES 2.x and 5.x but this will need to be specifically enabled.
+The Magento 2.3.4 update adds support for Elasticsearch (ES) 6.8.x and 7.x.x.
```
Comment (keharper) [inline src/guides/v2.3/config-guide/elasticsearch/es-downgrade.md]:
```suggestion
The Magento 2.3.5 update adds support for Elasticsearch (ES) 6.8.x and 7.x.x.
```

File: src/guides/v2.3/config-guide/elasticsearch/es-overview.md [hunk 2]
```diff
@@ -32,18 +32,14 @@ Using [Elasticsearch][] as your [catalog](https://glossary.magento.com/catalog)
 
 ### Supported versions {#es-spt-versions}
 
-{:.bs-callout-warning}
-Magento 2.3.1 adds support for Elasticsearch 6.x and it is enabled by default.
-Magento still provides connectivity for Elasticsearch 2.x and 5.x but these must be enabled in order to use these versions.
-Elasticsearch 2.x is still supported but strongly discouraged.
-If you must run Elasticsearch 2.x or 5.x with Magento 2.3.1, you must change the Elasticsearch php client.
+Magento 2.3.4 adds support for Elasticsearch 7.x.x (default) and 6.8.x.
```
Comment (keharper) [inline src/guides/v2.3/config-guide/elasticsearch/es-overview.md]:
```suggestion
Magento 2.3.5 adds support for Elasticsearch 7.x.x (default) and 6.8.x.
```

File: src/guides/v2.3/config-guide/elasticsearch/es-overview.md [hunk 3]
```diff
@@ -32,18 +32,14 @@ Using [Elasticsearch][] as your [catalog](https://glossary.magento.com/catalog)
...
+Both ES 2.x and 5.x are [End of Life][] and are no longer supported in Magento.
 Follow the instructions in [Change Elasticsearch Client][].
 
-{{site.data.var.ee}} version 2.3.x supports the following Elasticsearch versions:
+{{site.data.var.ee}} version 2.3.4 supports the following Elasticsearch versions:
```
Comment (keharper) [inline src/guides/v2.3/config-guide/elasticsearch/es-overview.md]:
```suggestion
{{site.data.var.ee}} version 2.3.5 supports the following Elasticsearch versions:
```

File: src/_includes/config/es-elasticsearch-magento.md [hunk 4]
```diff
@@ -2,10 +2,8 @@ This section discusses the minimum settings you must choose to test Elasticsearc
 For additional details about configuring Elasticsearch, see the [{{site.data.var.ee}} User Guide](http://docs.magento.com/m2/ee/user_guide/catalog/search-elasticsearch.html).
 
 {:.bs-callout-warning}
-Magento 2.3.1 adds support for Elasticsearch 6.x, and it is enabled by default.
-Magento still provides modules for Elasticsearch 2.x and 5.x, but these must be enabled in order to use these versions.
-Elasticsearch 2.x is still available but strongly discouraged. Versions 2.x and 5.x are [End of Life](https://www.elastic.co/support/eol).
-If you must run Elasticsearch 2.x or 5.x with Magento 2.3.1, you must downgrade the Elasticsearch client to an older version.
+Magento 2.3.5 adds support for Elasticsearch 7.x.x and 6.8.x.
```
Comment (keharper) [inline src/_includes/config/es-elasticsearch-magento.md]: Is there somewhere that lists all the versions Magento supports, vs what versions this particular release adds?

**Comment by dobooth** [comment]
running tests

**Pull request merged**